.github/workflows/ci.yml

```diff
@@ -218,7 +218,7 @@ jobs:
     name: Camunda docker tests
     needs: [ detect-changes ]
     runs-on: ubuntu-latest
-    timeout-minutes: 20
+    timeout-minutes: 40
```

Comment:
Why is this needed? We must avoid these kinds of changes to the `ci.yml` at all costs.